src/ui/FRWComponent/LandingPages/ImportComponents/Googledrive.tsx

```diff
@@ -42,9 +43,12 @@ const Googledrive = ({ setErrorMessage, setShowError }) => {
 
     try {
       const accounts = await wallets.loadBackupAccounts();
+
+      localStorage.setItem('backupAccounts', JSON.stringify(accounts));
+
       if (accounts.length > 0) {
         history.push({
-          pathname: '/import/google',
+          pathname: '/add/google',
           state: {
             accounts: accounts,
           },
```

Comment:
I'd prefer that re-usuable components don't navigate to routes. It makes them hard to re-use and keep track of routes. This route is now wrong, for example. Can you figure out a way to pass back to the parent so that the view containing this component can navigate to the route?